README.md

```diff
@@ -17,6 +17,16 @@ choose a secondary color like "cardinal" or "digital green".
 
 See <https://identity.stanford.edu/design-elements/color/web/>
 
+## HTML head link
+
+```html
+  <link rel="icon" href="https://cdn.jsdelivr.net/gh/sul-dlss/component-library@v2024-09-04/styles/icon.png" type="image/png">
+  <link rel="icon" href="https://cdn.jsdelivr.net/gh/sul-dlss/component-library@v2024-09-04/styles/icon.svg" type="image/svg+xml">
```

Comment:
Won't this not work? Since it won't be there until the new release?

Reply:
Right, it's just a sample layout until we get this merged and have a real release.